draft-ietf-tls-tls13.md

```diff
@@ -2242,12 +2267,15 @@ extension type (Supported Group Extension):
 The client MUST supply a "supported_groups" extension containing at
 least one group for each key exchange algorithm (currently
 DHE and ECDHE) for which it offers a cipher suite.
-If the client does not supply a "supported_groups" extension with a
-compatible group, the server MUST NOT negotiate a cipher suite of the
-relevant type.  For instance, if a client supplies only ECDHE groups,
-the server MUST NOT negotiate finite field Diffie-Hellman.  If no
-acceptable group can be selected across all cipher suites, then the
-server MUST generate a fatal "handshake_failure" alert.
+Servers receiving a TLS 1.3 ClientHello without this extension yet
+offering either DHE or ECDHE cipher suites MUST send a
+"missing_extension" alert message and close the connection.
```

Comment:
GitHub can't even find its own comments. The email links to nowhere, though if you manually find the right collapsed outdated diff in the closed PR it's in there. I'm normally all for discussion in the bug, but not in closed PRs with this system. New issue/PR and/or list for this sort of thing, please, if just so I can keep track of it. ;)